src/mame/drivers/hp9k_3xx.cpp

```diff
@@ -599,10 +460,6 @@ ROM_START( hp9k310 )
 	ROM_LOAD16_BYTE( "1818-3771.bin", 0x000001, 0x008000, CRC(b9e4e3ad) SHA1(ed6f1fad94a15d95362701dbe124b52877fc3ec4) )
 	ROM_LOAD16_BYTE( "1818-3772.bin", 0x000000, 0x008000, CRC(a3665919) SHA1(ec1bc7e5b7990a1b09af947a06401e8ed3cb0516) )
 
-	ROM_REGION( 0x800, IOCPU_TAG, 0 )
-	ROM_LOAD( "1820-4784_1.bin", 0x000000, 0x000800, CRC(e929044a) SHA1(90849a10bdb8c6e38e73ce027c9c0ad8b3956b1b) )
-	ROM_LOAD( "1820-4784_2.bin", 0x000000, 0x000800, CRC(8defcf50) SHA1(d3abfea468a43db7c2369500a3e390e77a8e22e6) )
```

Comment:
Is `1820-4784_2.bin` a legitimate alternate 8042 program? If so, it should be kept as an alternate BIOS for the `human_interface` device, not simply removed from MAME.

Reply:
It's moved to human_interface.cpp...

Reply:
It's not - if you look at the file it looks like a repeating pattern. It's definitely not a valid firmware.